sirepo/package_data/static/js/sirepo-components.js

```diff
@@ -1741,35 +1745,21 @@ SIREPO.app.directive('appHeaderBrand', function() {
             '<div class="navbar-header">',
               '<a class="navbar-brand" href="/en/landing.html"><img style="width: 40px; margin-top: -10px;" src="/static/img/sirepo.gif" alt="RadiaSoft"></a>',
               '<div class="navbar-brand">',
-                '<span data-ng-if="! hasAppLink()">', appInfo.longName, '</span>',
-                '<a data-ng-if="hasAppLink()" data-ng-href="{{ appUrl || nav.sectionURL(\'simulations\') }}">',
-                  '<span class="hidden-md hidden-sm">',
-                    appInfo.longName,
-                  '</span>',
+                appUrl ? '<a data-ng-href="' +  appUrl + '">' : '',
```

Comment:
Wrap this in a function instead of the stateful approach:

```js
function appUrl() {
    var res = [
                '<span data-ng-if="! hasAppLink()">', appInfo.longName, '</span>',
                '<a data-ng-if="hasAppLink()" data-ng-href="{{ appUrl || nav.sectionURL(\'simulations\') }}">',
                  '<span class="hidden-md hidden-sm">',
                    appInfo.longName,
                  '</span>',
     ].join('');
     var u = {
        elegant: '/en/particle-accelerators.html',
        srw: '/en/xray-beamlines.html'
    }[SIREPO.APP_NAME];
    if (!u) {
        return res;
    }
    return '<a data-ng-href="' + u + '">' + res + '</a>';
}
```